elements/lisk-dpos/src/dpos.ts

```diff
@@ -196,7 +163,10 @@ export class Dpos {
 			delegateActiveRoundLimit,
 		);
 
-		return this.rounds.calcRoundStartHeight(activeRounds);
+		const minBootstrapHeight = this.rounds.lastHeightBootstrap() + 1;
```

Comment:
`minBootstrapHeight` is a bit misleading, because it is the first height that is not in the bootstrapping phase